### DISCUSSION TITLE
Is there a way to save progress on the timeline?

### DISCUSSION THREAD

**Discussion by PoliyasRoden** [dialogic-godot/dialogic#2086] [Q&A]
hello. Thanks to Dialog, I am making a game with my inexperienced skills. thank you

Is there a way to save progress on the timeline?

I would like to display an option scene when I press a specific button on the timeline. (change_scene_to_packed()) If you return to the dialog after setting the option, it starts again from the beginning...

Various methods are welcome. please.

**Comment by CakeVR** [comment]
Hello, yes, you can save the progress of your timeline.

Take a look at this: https://dialogic-docs.coppolaemilio.com/saving-and-loading.html
